inference-engine/src/gna_plugin/backend/am_intel_dnn.cpp

```diff
@@ -159,8 +160,13 @@ void GNAPluginNS::backend::AMIntelDNN::InitConvolutional1DComponentPrivate(intel
     comp.num_bytes_per_output = num_bytes_per_output;
     comp.operation = kDnnConvolutional1dOp;
     comp.macro_operation = kDnnMacroOpNone;
-    comp.orientation_in = kDnnNonInterleavedOrientation;
-    comp.orientation_out = kDnnNonInterleavedOrientation;
+    if (layout == 2) {
```

Comment:
The code will be more legible if Layout::NHWC would be used here instead of a number.